docker/docker-compose.yml

```diff
@@ -14,7 +14,7 @@ services:
       - TEST_ENV=true
       - PYTHONUNBUFFERED=0
     ports:
-      - $VIRTUAL_HOST_EXPOSE_PORT:$VIRTUAL_HOST_PORT
+      - 8000:8000
```

Comment:
It would be nice to leave `VIRTUAL_HOST_EXPOSE_PORT`, because 8000 may be acquired

Reply:
@artemiy312 , y, it's true. We only have confusing with `VIRTUAL_HOST_PORT` vs `VIRTUAL_HOST_EXPOSE_PORT`.
So, i'll call it `VIRTUAL_HOST_DEV_PORT`